Sources/SwiftDocC/Model/DocumentationNode.swift

```diff
@@ -210,14 +211,9 @@ public struct DocumentationNode {
         
         self.availableSourceLanguages = reference.sourceLanguages
         
-        let extendedModuleVariants = DocumentationDataVariants(
-            symbolData: unifiedSymbol.mixins,
-            platformName: platformName
-        ) { mixins in
-            return mixins.getValueIfPresent(
-                for: SymbolGraph.Symbol.Swift.Extension.self
-            )?.extendedModule
-        }
+        let extendedModule: String? = unifiedSymbol.mixins
+            .first(where: { $0.0.platform == platformName })?.value
+            .getValueIfPresent(for: SymbolGraph.Symbol.Swift.Extension.self)?.extendedModule
```

Comment:
Not sure I understand this change, can you elaborate? Should we not traverse the `mixins` for each language variant the symbol is available in?

Reply:
From what I understand, module names are required to be the same in all languages. The `Symbol` semantic no longer stores variants of module related information so this computes a single language agnostic value.

Reply:
I think that makes sense, because the `UnifiedSymbolGraph` logic relies on the symbol graph files for each language variant of a module to be the same, right @QuietMisdreavus?